components/common/IdentityForm.vue

```diff
@@ -227,6 +242,26 @@ const {
 const isConfirmModalActive = ref(false)
 const isLoaderModalVisible = ref(false)
 const transactionValue = ref('')
+const menus = availablePrefixes().filter(
+  (menu) =>
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
well I tried to figure out issue but I didn't found any bugs with this line of code can you explain me more about this @vikiival

Reply:
done fixed if isAssetHub Create or Update button will get disabled
I tried a different way :)

Reply:
code is still present

Reply:
> code is still present

yeah and it should be there vikival wanted to disable that create identity button because when users select ahp or ahk from nav it used to activate Create button and to disable that he recommended me a way to filter then disable button using javascript but I tried a different way using isAssetHub disabled within Neobutton

in this code it is disabling asset hub and moonriver, moonbeam in the selection
so we are just keep disabling them so users don't get stuck around disabled button after selecting one of these chains :)

Reply:
> code is still present

I'm thinking about user experience if you think we should remove it I'm ready
as you know currently ahk and ahp don't support on-chain identity so adding chain over there won't be good idea once both of them supports identity then we can enable these chains